packages/org-unit-dialog/src/OrgUnitSelector.js

```diff
@@ -99,49 +101,70 @@ class OrgUnitSelector extends Component {
     };
 
     renderOptionsPanel = () => (
-        <Grid
-            spacing={8}
-            style={styles.footer.index}
-            container
-        >
-            <Grid item xs={4}>
-                <FormControl style={{ width: '100%' }}>
-                    <InputLabel htmlFor="level-select">{i18n.t('Level')}</InputLabel>
-                    <Select
-                        value={this.props.level}
-                        onChange={this.props.onLevelChange}
-                        input={<Input id="level-select" />}
-                        renderValue={this.renderLevelOptions}
-                        disabled={this.props.userOrgUnits.length > 0}
-                        multiple
-                        fullWidth
-                    >
-                        {this.props.levelOptions.map(option => (
-                            <MenuItem key={option.id} value={option.id}>{option.displayName}</MenuItem>
-                        ))}
-                    </Select>
-                </FormControl>
-            </Grid>
-            <Grid item xs={4}>
-                <FormControl style={{ width: '100%' }}>
-                    <InputLabel htmlFor="group">{i18n.t('Group')}</InputLabel>
-                    <Select
-                        value={this.props.group}
-                        onChange={this.props.onGroupChange}
-                        input={<Input name="group" id="group" />}
-                        renderValue={this.renderGroupOptions}
-                        disabled={this.props.userOrgUnits.length > 0}
-                        multiple
-                        displayEmpty
-                        fullWidth
-                    >
-                        {this.props.groupOptions.map(option => (
-                            <MenuItem key={option.id} value={option.id}>{option.displayName}</MenuItem>
-                        ))}
-                    </Select>
-                </FormControl>
+        <div style={styles.footer.index}>
+            <div style={styles.footer.tooltipContainer}>
+                {this.props.selected.length > 0 && (
+                    <div style={styles.footer.tooltip}>
+                        {this.props.selected.length} selected.
+                        <a
+                            onClick={this.props.onDeselectAllClick}
+                            style={styles.footer.tooltip.link}
+                            // eslint-disable-next-line
+                            href="javascript:void(0)"
+                        >
+                            Deselect all
+                        </a>
+                    </div>
+                )}
+            </div>
+            <Grid style={styles.footer.gridContainer} container>
+                <Grid
```

Comment:
Even though this code was not part of this PR, you could remove some code duplication (of the <Grid components) by adding a stateless component like so (not sure if GridControl is the best name):
```
<GridControl
                    title={i18n.t('Level')}
                    id="level-select"
                    value={this.props.level}
                    onChange={this.props.onLevelChange}
                    options={this.props.levelOptions}
                    disabled={this.props.userOrgUnits.length > 0}
                    renderValue={this.renderLevelOptions}
                    multiple
                />```

And 
```
const GridControl = ({id, title, options, ...props}) => {
    return (<Grid
        item
        xs={4}
        style={styles.footer.gridContainer.gridItem}
    >
        <FormControl style={{ width: '100%' }}>
            <InputLabel htmlFor={id}>{title}</InputLabel>
            <Select
                input={<Input id={id} />}
                fullWidth
                {...props}
            >
                {options.map(option => (
                    <MenuItem key={option.id} value={option.id}>{option.displayName}</MenuItem>
                ))}
            </Select>
        </FormControl>
    </Grid>);
}
```

Reply:
And the component (might not display correctly here):

``````
const GridControl = ({id, title, options, ...props}) => {
    return (<Grid
        item
        xs={4}
        style={styles.footer.gridContainer.gridItem}
    >
        <FormControl style={{ width: '100%' }}>
            <InputLabel htmlFor={id}>{title}</InputLabel>
            <Select
                input={<Input id={id} />}
                fullWidth
                {...props}
            >
                {options.map(option => (
                    <MenuItem key={option.id} value={option.id}>{option.displayName}</MenuItem>
                ))}
            </Select>
        </FormControl>
    </Grid>);
}```